README.md

```diff
@@ -84,13 +84,27 @@ For more models, go to [**tensorflow/swift-models**](https://github.com/tensorfl
 
 ### Building and testing
 
+#### Building using swift-package-manager (swift-build)
+
 ```
 $ swift build
 ```
 ```
 $ swift test
 ```
 
+#### Building using CMake
+
+Note that this requires CMake 3.15.1 or newer.  In-tree builds are not
+supported.  The following assumes that the intermediates should be placed in
+`/BinaryCache/tensorflow-swift-apis` and that the sources have been checked out
```

Comment:
To build using CMake, is it required for `tensorflow-swift-apis` sources to exist at `/SourceCache/tensorflow-swift-apis`?

Can sources and binary intermediates be placed elsewhere? It's not clear from the current instructions.

Reply:
They can be placed anywhere.  I can reword the "assumes that the intermediates should placed in ... and that the sources have been checked out at ..."